lib/API.js

```diff
@@ -851,9 +851,9 @@ API.prototype._startJson = function(file, opts, action, pipe, cb) {
   appConf.forEach(function(app) {
     if (opts.only && opts.only != app.name)
       return false;
-    if (opts.watch && opts.watch === true)
+    if (opts.watch && opts.watch === true && !app.watch)
       app.watch = true;
-    if (opts.ignore_watch)
+    if (opts.ignore_watch && !app.ignore_watch)
```

Comment:
@soyuka Thank you for your feedback! Have a look at 53c814d and 86af1f5. I'm not sure why the Node.js 0.10 was not successful. I think it's a timing issue. Can you restart the Node.js 0.10 test?